Unescape urls extract email v2 (#14535)
demisto/content: Packs/CommonTypes/ReleaseNotes/3_1_12.md

#### Indicator Types

> - **emailRep** - added to regex emails with [@] as valid .

Reviewer: ```suggestion
- **emailRep** - The regex now recognizes emails with `[@]` as valid.
```
